### PR TITLE
A-SaveFile

### PR DESCRIPTION
**Pull request by rexkoh425**
Add support for saving transaction , customer , car data to files. Update DG. Fixes https://github.com/AY2425S1-CS2113-T11-3/tp/issues/101 https://github.com/AY2425S1-CS2113-T11-3/tp/issues/89 https://github.com/AY2425S1-CS2113-T11-3/tp/issues/90 https://github.com/AY2425S1-CS2113-T11-3/tp/issues/91 https://github.com/AY2425S1-CS2113-T11-3/tp/issues/92

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -42,6 +42,7 @@ checkstyle {
 }
 
 run{
+    enableAssertions = true
```
Comment (CT9ARyan) [inline build.gradle:45]: Maybe you can remove this line?? There is already one below

**Review by SemiColonKen — COMMENTED** [review]
LGTM!

File: src/main/java/customer/Customer.java [hunk 2]
```diff
@@ -5,11 +5,12 @@
  */
 public class Customer {
 
+    public static final int NUMBER_OF_PARAMETERS = 3;
     private String username;
     private int age;
-    private int contactNumber;
+    private String contactNumber;
```
Comment (CT9ARyan) [inline src/main/java/customer/Customer.java:11]: Good 👍

**Review by CT9ARyan — COMMENTED** [review]
LGTM 👍

**Pull request merged**